Extension/src/LanguageServer/configurations.ts

```diff
@@ -1377,6 +1377,11 @@ export class CppProperties {
             result = result.replace(/\*/g, "");
         }
 
+        // Make sure all paths result to an absolute path
+        if (!path.isAbsolute(result)) {
+            result = path.join(this.rootUri?.fsPath as string, result);
```

Comment:
This will fail with "'The "path" argument must be of type string. Received undefined" if this.rootUri is undefined. I'm not sure if that's possible but can you safeguard against that?

Reply:
I think this block needs to be moved up into the block that checks `if (this.rootUri)`.

Reply:
I think it'd be better to keep it at the bottom just to make sure